### DISCUSSION TITLE
Does this library support the new RN architecture?

### DISCUSSION THREAD

**Discussion by bitcrumb** [react-native-async-storage/async-storage#968] [Q&A]
Since this library is mentioned on the https://expo.dev/ website. And SDK 48 announced support for the new RN architeture. Can I assume this also applies for this lib?

**Comment by bitcrumb** [comment]
Doesn't look like it?
https://github.com/react-native-async-storage/async-storage/pull/910

**Reply by tido64** [reply]
Not at the moment. Someone attempted it then kinda just abandoned it. This PR will at least make it compatible with TM though: https://github.com/react-native-async-storage/async-storage/pull/965
